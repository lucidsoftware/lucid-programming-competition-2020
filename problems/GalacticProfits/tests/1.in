6
3
Dusty_Logistics
200 700 300 200 600 600
Red_Rovers
100 800 400 600 700 300
Methane_Holdings_Co
25 20 65 86 30 67
