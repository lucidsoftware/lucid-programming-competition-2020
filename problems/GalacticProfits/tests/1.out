Max: 42698
Min: -9438
